core/new-gui/src/app/common/service/user/google.service.ts

```diff
@@ -1,9 +1,8 @@
 import { Injectable } from "@angular/core";
 import { Subject } from "rxjs";
 import { HttpClient } from "@angular/common/http";
-import { AppSettings } from "../../common/app-setting";
+import { AppSettings } from "../../app-setting";
 declare var window: any;
-
 export interface credentialRes {
```

Comment:
if this interface is defined by us, can we follow the naming convention to name it as `CredentialResponse`?

Reply:
Renamed.